cronet/src/main/java/io/grpc/cronet/CronetChannelBuilder.java

```diff
@@ -132,7 +132,7 @@ protected ManagedChannelBuilder<?> delegate() {
    * Sets the maximum message size allowed to be received on the channel. If not called,
    * defaults to {@link io.grpc.internal.GrpcUtil#DEFAULT_MAX_MESSAGE_SIZE}.
    */
-  public final CronetChannelBuilder maxMessageSize(int maxMessageSize) {
+  public CronetChannelBuilder maxMessageSize(int maxMessageSize) {
```

Comment:
CronetChannelBuilder Class is final. Methods inherit final from the class, removing explicitly call out final.
`public final class CronetChannelBuilder extends ForwardingChannelBuilder2<CronetChannelBuilder> {`